### PR TITLE
Feature/connect spot tile rebased

### PR DESCRIPTION
**Pull request by PawelAP**
Big refactor

### CODE REVIEW

File: src/client/config/openfin/demo.app.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "splashScreenImage": "https://web-demo.adaptivecluster.com/images/splash-screen.jpg",
     "startup_app": {
         "name": "Reactive Trader Cloud",
-        "url": "https://web-demo.adaptivecluster.com",
+        "url": "http://web-dev.adaptivecluster.com/",
```
Comment (tdeheurles) [inline src/client/config/openfin/demo.app.json]: why these changes ?
1) https ==> http
2) demo ==> dev

Comment (PawelAP) [reply src/client/config/openfin/demo.app.json]: I don't know and the other guys are on holidays
